regression/validate-trace-xml-schema/validate-xsd/src/main/java/com/diffblue/validatexsd/Main.java

```diff
@@ -0,0 +1,50 @@
+package com.diffblue.validatexsd;
+
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXParseException;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Parameters;
+
+import javax.xml.XMLConstants;
+import javax.xml.transform.sax.SAXSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+import java.io.FileInputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.concurrent.Callable;
+
+@Command(name = "validate-xsd", mixinStandardHelpOptions = true, version = "0.1",
+  description = "tells you if given XML (read from stdin) matches a given XSD spec")
+public class Main implements Callable<Integer> {
+  @Parameters(index = "0")
+  private Path xsdSchemaFilePath;
+
+  @Override
+  public Integer call() throws Exception {
```

Comment:
:no_entry_sign: Sorry to be a pain, but can we have 4 simple tests just to check the errors get thrown correctly:

 - no schema
 - invalid schema
 - invalid xml
 - valid xml

Hopefully these don't take long to write.